manager/integration/tests/test_basic.py

```diff
@@ -5522,7 +5531,52 @@ def test_backuptarget_invalid(): # NOQA
     Then
     - Backup will be failed and the backup state is Error.
     """
-    pass
+    def delete_all_backups():
+        # Use k8s api to delete all backup including backup in error state
+        api = get_custom_object_api_client()
+        backups = api.list_namespaced_custom_object("longhorn.io",
+                                                    "v1beta2",
+                                                    "longhorn-system",
+                                                    "backups")
+        for backup in backups['items']:
+            api.delete_namespaced_custom_object("longhorn.io",
+                                                "v1beta2",
+                                                "longhorn-system",
+                                                "backups",
+                                                backup['metadata']['name'])
+
+    request.addfinalizer(delete_all_backups)
```

Comment:
Perhaps we could consider relocating the `delete_all_backups` function to `common.py` and then executing it within the `cleanup_client()` function. Do you have any thoughts or feedback on this suggestion?

Reply:
Updated. I reorganized `delete_all_backups` into  fixure `backupstore_invalid` because only fixture `backupstore_invalid` will create backup in error state. (All other backups have corresponding backup volumes will clean up in `backupstore_cleanup` at `backupstore.py`)

test [result](https://ci.longhorn.io/job/private/job/longhorn-tests-regression/4924/)